Supplements/Virgo/portalContext-security.xml

```diff
@@ -22,19 +22,29 @@
 
 
 <beans:beans xmlns="http://www.springframework.org/schema/security"
-             xmlns:beans="http://www.springframework.org/schema/beans" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+             xmlns:context="http://www.springframework.org/schema/context"
+             xmlns:beans="http://www.springframework.org/schema/beans"
+             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
              xsi:schemaLocation="
+        http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context.xsd
         http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
         http://www.springframework.org/schema/security http://www.springframework.org/schema/security/spring-security-3.1.xsd">
 
-    <!-- Uncomment to support new user registration and reset password.
+    <!-- Enable auto-wiring -->
+    <context:annotation-config/>
+    <context:component-scan base-package="org.springframework.security.saml"/>
+
+    <beans:bean id="loginPageFilter" class="org.apromore.portal.security.LoginRedirectKeycloakFilter" />
+
+    <!-- Setup for the Servlets used in new user regisration and reset password.
     <beans:bean id="resetPassword" class="org.apromore.portal.servlet.ResetPasswordHttpServletRequestHandler" />
     <beans:bean id="newUserRegistration" class="org.apromore.portal.servlet.NewUserRegistrationHttpServletRequestHandler" />
 
     <http pattern="/register/**" security="none"/>
     -->
 
     <!-- Http Security Setup -->
+    <http pattern="/applets/**" security="none"/>
```

Comment:
This is very strange.  A merge artifact?

Reply:
Removed old checkout line / merge atefact [DONE].